hw/top_earlgrey/rdc/chip_earlgrey_asic.env

```diff
@@ -22,3 +22,9 @@ set_constant -value 2'b00 top_earlgrey.u_pinmux_aon.u_pinmux_strap_sampling.tap_
 
 # Define POK as reset
 create_reset -interval 10 -waveform AON_CLK u_ast.vcaon_pok
+
+# Define SPI_DEVICE CSB as reset
+create_reset -interval 10 -high -waveform SPI_DEV_CLK top_earlgrey.u_spi_device.rst_csb_buf
```

Comment:
does the `SPI_DEV_CLK` here mean the reset is related to that clock?

Reply:
Yes. The `-interval` is based on the CLK.